### ISSUE TITLE
HTML elements with role="combobox" are missing aria attributes

### ISSUE THREAD

**Issue by Bratelion** [downshift-js/downshift#1336]
- `downshift` version: ^6.1.7
- `node` version: 14.17.3
- `yarn` version: 1.22.10

**What you did**: Code uses cypress for end-to-end testing with included package [`cypress-axe`](https://www.npmjs.com/package/cypress-axe) to detect a11y violations, as well as browser add-on [`axe DevTools`](https://www.deque.com/axe/devtools/)

**What happened**: HTML Elements with `role="combobox"` are missing required attributes

![Axe browser addon check](https://user-images.githubusercontent.com/19559116/142217717-75615aef-0e16-4fcd-a575-cf353dcbf69b.png)
![Cypress-axe check](https://user-images.githubusercontent.com/19559116/142218189-7485aff0-823e-4233-9eb7-e0d2c8c14a37.png)

**Problem description**: Per the [`documentation`](https://www.w3.org/TR/wai-aria-1.1/#combobox), for elements with `combobox` role, "When the combobox is expanded, authors MUST set aria-controls on the textbox element to a value that refers to the combobox popup element."

**Suggested solution**: Since both `cypress-axe` and `axe DevTools` are triggered in both cases, with expanded or not combobox, with accessibility flag `aria-required-attribute`, add the prop `ariaControls : PropTypes: string` to Dropshift Component, that developers can set accordingly.

**Comment by silviuaavram** [comment]
Hi @Bratelion! I don't think the changes are correct.

The combobox div should have the aria-owns pointing to the menu element. The input should have the aria-controls pointing to the menu element. And we have this, but these attributes are not added to the elements if the combobox is collapsed. However, we fixed this in `useCombobox` which I recommend using instead of `Downshift`.

To properly check that this is fixed, please run your axe tool on https://www.downshift-js.com/use-combobox. There is also the example for Downshift there as well, to check that the tools shows the issues.

Consequently, shift to useCombobox and it should work as expected. Let me know if I can help further. Thanks!
